app/src/main/scala/org/alephium/explorer/persistence/queries/HashrateQueries.scala

```diff
@@ -33,33 +34,42 @@ trait HashrateQueries extends CustomTypes {
   def getHashratesQuery(from: TimeStamp,
                         to: TimeStamp,
                         interval: Int): DBActionSR[(TimeStamp, BigDecimal)] = {
-    val fromInstant = Instant.ofEpochMilli(from.millis)
-    val toInstant   = Instant.ofEpochMilli(to.millis)
 
     sql"""
         SELECT timestamp, value
         FROM hashrates
-        WHERE interval_type = #$interval
-        AND timestamp >= '#${fromInstant.toString}'
-        AND timestamp <= '#${toInstant.toString}'
+        WHERE interval_type = $interval
+        AND timestamp >= $from
+        AND timestamp <= $to
         ORDER BY timestamp
       """.as[(TimeStamp, BigDecimal)]
   }
 
   def computeHashratesAndInsert(from: TimeStamp, intervalType: Int): DBActionW[Int] = {
-    val hashrates = (intervalType match {
-      case 0 => compute10MinutesHashrateRawString(from)
-      case 1 => computeHourlyHashrateRawString(from)
-      case 2 => computeDailyHashrateRawString(from)
-      case _ => "()"
-    })
-
-    sqlu"""
-      INSERT INTO hashrates (timestamp, value, interval_type)
-      (#$hashrates)
-      ON CONFLICT (timestamp, interval_type) DO UPDATE
-      SET value = EXCLUDED.value
-    """
+    if (intervalType >= 0 || intervalType <= 2) {
+      val dateGroup = (intervalType match {
+        case 0 => tenMinutesQuery
```

Comment:
I am wondering if `10 minutes` average makes sense now

Reply:
@tdroxler Let's remove 10 minutes for now. We both think it's not helpful

Reply:
done